db/models.py

```diff
@@ -45,3 +53,62 @@ class MovieSession(models.Model):
 
     def __str__(self):
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey("User", on_delete=models.CASCADE)
```

Comment:
Taras is right, It is better to use either `settings.AUTH_USER_MODEL` or `get_user_model()`.